package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "defaults": {
     "omniSharp": "1.35.3",
-    "razor": "1.0.0-alpha3-5.0.0-preview.7.20309"
+    "razor": "1.0.0-alpha3-5.0.0-preview.8.20359.1"
```

Comment:
Lets take this opportunity to go from `1.0.0-alpha3-5.0.0-preview.7.20309` -> `5.0.0-preview.7.20309`